usr/src/cmd/zfs/Makefile

```diff
@@ -32,6 +33,7 @@ POFILES=	zfs_main.po zfs_iter.po
 POFILE=		zfs.po
 
 include ../Makefile.cmd
+#include ../Makefile.cmd.64
```

Comment:
it is example how to build 64bit zfs and use it.
if it is not applicable - i'll remove it, or we can switch to 64bit - what you are prefer?